docs/advanced-usage/packaging-for-automated-evaluation.md

```diff
@@ -214,6 +214,20 @@ cases and our models.
 
 ## Appendix
 
+### Testing evaluator locally
+
+You can verify the evaluator Docker image by running it locally:
+
+```
+docker run --rm \
+  -e KOLENA_TEST_SUITE_NAME="my test suite" \
+  -e KOLENA_TEST_SUITE_VERSION=3 \
+  -e KOLENA_MODEL_NAME="keypoint detection model" \
```

Comment:
This wouldn't need to be an existing model, correct? Maybe something that covers the intent of the direct run?
```suggestion
  -e KOLENA_MODEL_NAME="verify-evaluator-image" \
```

Reply:
To me, the suggested wording change doesn't necessarily convey to users this value is not used (if it is, can just drop this variable). On the other hand, the command is not restricted to verify build correctness. I think it'd be useful that users can use it to actually complete a metrics re-computation.

Reply:
Nit: I always like to include the string `example`, e.g. `example keypoint detection model`, to be clear that this is not an important string